package.json

```diff
@@ -1,123 +1,19 @@
 {
-  "name": "@ember/test-waiters",
-  "version": "3.0.2",
-  "description": "Allow tests to be aware of addon / application asynchrony.",
-  "keywords": [
-    "ember-addon"
-  ],
-  "repository": "https://github.com/emberjs/ember-test-waiters",
-  "license": "MIT",
-  "contributors": [
-    "Steve Calvert <steve.calvert@gmail.com>",
-    "Robert Jackson <me@rwjblue.com>"
-  ],
-  "types": "index.d.ts",
-  "directories": {
-    "doc": "doc",
-    "test": "tests"
-  },
+  "name": "root",
+  "private": true,
+  "version": "0.0.0",
   "scripts": {
-    "build": "ember build",
-    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel \"lint:!(fix)\"",
-    "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
-    "lint:js": "eslint --ext ts,js .",
-    "lint:js:fix": "eslint --ext ts,js . --fix",
-    "lint:ts": "tsc -p tsconfig.json --noEmit",
-    "prepack": "npm-run-all prepack:*",
-    "prepack:d.ts": "ember ts:precompile",
-    "prepack:addon": "babel --extensions '.ts' --presets @babel/preset-typescript addon --out-dir addon --ignore '**/*.d.ts'",
-    "postpack": "ember ts:clean",
-    "start": "ember serve",
-    "test": "ember test",
-    "test:ember": "ember test",
-    "test:ember-compatibility": "ember try:each",
-    "test:prod": "ember test --environment=production"
+    "lint": "pnpm --filter '*' lint:fix",
+    "start": "concurrently 'npm:start:*' --restart-after 5000 --prefix-colors cyan,white,yellow",
+    "start:addon": "pnpm --filter @ember/test-waiters start -- --no-watch.clearScreen",
+    "start:test-app": "pnpm --filter test-app start",
```

Comment:
technically this is not yet ready, I believe we can keep as it will appear in next PR.

Reply:
yeep